src/Contract/Test/Plutip.purs

```diff
@@ -41,6 +47,13 @@ testPlutipContracts
   -> MoteT Aff (Aff Unit) Aff Unit
 testPlutipContracts = Server.testPlutipContracts
 
+-- | Run `ContractTestPlan` tests in a single Plutip instance.
+runPlutipTestPlan
+  :: PlutipConfig
+  -> Server.ContractTestPlan
+  -> MoteT Aff (Aff Unit) Aff Unit
+runPlutipTestPlan = Server.runPlutipTestPlan
```

Comment:
Looks like a plain re-export?

Reply:
I just followed the exporting of `testPlutipContracts` as en example, should that just be a re-export as well?